### PR TITLE
feat: added Product Details Page

### PR DESCRIPTION
**Pull request by DevShobhit**
## Related Issue #613 

Closes: #613  

#### Describe the changes you've made ✍
- Created new component `ProductDetails.jsx` 
- Created `ProductDetails.css`
- Changed `DetailedProduct` Component
- Added Route for DetailedProduct inside `App.jsx`

#### Screenshots

image
image

### CODE REVIEW

**Comment by IAmTamal** [comment]
@DevShobhit , please fix the PR title. Take a look and read at what the bot says !

File: src/components/shop/ProductDetails.jsx
Comment (IAmTamal) [inline src/components/shop/ProductDetails.jsx:1]: Can we have the `svg` in separate files ?

**Review by IAmTamal — CHANGES_REQUESTED** [review]
Rest looks good to me, however, can we add a `<Link>` to the products on the cards so that they take me to the details page ?

File: src/components/shop/ProductDetails.jsx
Comment (DevShobhit) [reply src/components/shop/ProductDetails.jsx:1]: Yes, that's a good idea

**Comment by DevShobhit** [comment]
> Rest looks good to me, however, can we add a `<Link>` to the products on the cards so that they take me to the details page ?

@IAmTamal We can add a button at the bottom of products card. As this would require changing the structure of product card, we can create a new issue for adding button at the bottom of products card and creating Api for fetching product details.

Should I proceed to make changes to product card and create Api for fetching product details with this PR?

**Pull request closed without merging**